script.service.hyperion-control/resources/__init__.py

```diff
@@ -0,0 +1 @@
+"""Hyperion control addon resources."""
```

Comment:
dummy init files are not needed in python3

Reply:
File removed